CMakeLists.txt

```diff
@@ -19,6 +19,9 @@ endif()
 
 project(hera)
 
+cable_configure_compiler()
+add_compile_options(-Wno-pedantic -Wno-sign-conversion)
```

Comment:
Does cable adds `-Wpedantic` so that we need to negative it here?

Any point keeping `-Wno-unknown-pragmas`?

Reply:
Can you remove `-Wno-sign-conversion` and update `evm2wasm`?

Reply:
The `-Wno-sign-conversion` is also needed in Hera.

Reply:
Tracked in #311.